examples/IMU/MadgwickVisualization/Processing/MadgwickVisualizationDraw/MadgwickVisualizationDraw.pde

```diff
@@ -0,0 +1,188 @@
+import processing.serial.*;
+import toxi.geom.*;
+import toxi.processing.*;
+
+// NOTE: requires ToxicLibs to be installed in order to run properly.
+// 1. Download from http://toxiclibs.org/downloads
+// 2. Extract into [userdir]/Processing/libraries
+//    (location may be different on Mac/Linux)
+// 3. Run and bask in awesomeness
+
+// The serial port
+Serial port;                         
+
+String message;
+
+float[] q = new float[4];
+Quaternion quat = new Quaternion(1, 0, 0, 0);
+// New line character in ASCII
+int newLine = 13; 
```

Comment:
```suggestion
const char newLine = 13; 
```
or
```suggestion
const char newLine = '\xD'; 
```
And the comment is wrong. The character at number 13 in ASCII is CR (carriage return).